.github/workflows/_buildpacks-release.yml

```diff
@@ -427,6 +431,7 @@ jobs:
             Update ${{ github.repository }} to v${{ needs.compile.outputs.version }}
 
             ${{ needs.compile.outputs.changelog }}
+          reviewers: ${{ inputs.reviewers }}
```

Comment:
The input will default to the empty string, which `peter-evans/create-pull-request` will handle fine:
https://github.com/peter-evans/create-pull-request/blob/58258f452b6c22066ed351398c99510be8470f5a/src/utils.ts#L16